File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/config/ErdException.java
package org.javamaster.invocationlab.admin.config;

import lombok.Getter;

/**
 * @author yudong
 * @date 2023/2/16
 */
@Getter
public class ErdException extends RuntimeException {
    private static final long serialVersionUID = 2362669725006014925L;
    private int code = 400;

    public ErdException(int code, String message) {
        this(message);
        this.code = code;
    }

    public ErdException(String message) {
        super(message);
    }

}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/config/ErdException.kt
package org.javamaster.invocationlab.admin.config

/**
 * @author yudong
 * @date 2023/2/16
 */

class ErdException(message: String) : RuntimeException(message) {
    var code = 400

    constructor(code: Int, message: String) : this(message) {
        this.code = code
    }
}
